static/js/ReaderApp.jsx

```diff
@@ -949,6 +949,7 @@ class ReaderApp extends Component {
         layoutTanakh:  "segmented",
         aliyotTorah:   "aliyotOff",
         vowels:        "all",
+        punctuationTalmud: "punctuationOn",
```

Comment:
Minor style point, same as above: for readability the values here can be aligned with spaces in each line.